Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -1616,13 +1623,21 @@ private bool PrepareFileAccessMonitoringCommon(LoggingContext loggingContext)
 
                     if (path != translatedPath)
                     {
-                        m_fileAccessManifest.AddScope(translatedPath, mask: m_excludeReportAccessMask, values: FileAccessPolicy.AllowAll | FileAccessPolicy.AllowRealInputTimestamps);
+                        AddUntrackedScopeToManifest(translatedPath);
                         Tracing.Logger.Log.TranslatePathInGlobalUnsafeUntrackedScopes(loggingContext, m_pip.SemiStableHash, m_pip.GetDescription(m_context), pathString);
                     }
-                }               
+                }
 
-                // untrack the original path
-                m_fileAccessManifest.AddScope(path, mask: m_excludeReportAccessMask, values: FileAccessPolicy.AllowAll | FileAccessPolicy.AllowRealInputTimestamps);
+                // Untrack the original path
+                AddUntrackedScopeToManifest(path);
+            }
+
+            if (!OperatingSystemHelper.IsUnixOS)

```

Comment:
Did you check with the mac team if they perhaps have a similar issue?

Reply:
Mac won't have this issue because
- it's Detours related,
- it's about a pip that also runs Detours during its execution.

---
In reply to: [303165462](https://github.com/microsoft/BuildXL/pull/589#discussion_r303165462) [](ancestors = 303165462)